app/views/exam_templates/_create_new_template.html.erb

```diff
@@ -4,18 +4,9 @@
   <%= form_for :create_template,
                url: course_assignment_exam_templates_path(@current_course, @assignment),
                html: { multipart: true, size: 1 } do |f| %>
```

Comment:
You can add a `remote: true` here to make this an AJAX request (which allows you to render JS instead of HTML from the controller route).

Reply:
I don't think that is required here. I need to refresh the page to update the list of templates on the side panel. This doesn't happen when I set `remote: true`

Reply:
@anoverga17 Yeah, so it is possible to refresh both lists (we do this for criteria/annotations), but it doesn't need to be in this PR. We can go with what you have!